src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpSyntaxTree.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.DotNet.Interactive.HttpRequest;
+
+/*
+
+Rough sketch, probably out of date:
```

Comment:
This is already out of date :) Perhaps now would be a good time to delete it since we have a shared understanding of what the tree looks like.